templates/corosync.conf.erb

```diff
@@ -37,7 +37,13 @@ totem {
 <% if @netmtu -%>
   netmtu:                              <%= @netmtu %>
 <% end -%>
-  secauth:                             <%= @enable_secauth ? 'on' : 'off' %>
+<% if @enable_secauth -%>
+  crypto_hash:                         <%= @crypto_hash %>
+  crypto_cipher:                       <%= @crypto_cipher %>
+<% else -%>
+  crypto_hash:                         none
+  crypto_cipher:                       none
+<% end -%>
```

Comment:
For compatibility reasons, switching away from `secauth` to `crypto_hash` and `crypto_cipher` requires corosync >= 2.0; `secauth` only works for corosync <= 3.0.

Might require a major version bump to a breaking API change.

Reply:
That's a fair point. I was originally considering adding logic to detect whether this was Corosync 2.x and choosing the value based on this setting.

Reply:
I've made this change in #460 which separates the first commit in this PR (REFERENCE.md & the secauth changes) into it's own branch.